### ISSUE TITLE
Donate (streamlined): Display full amount if agreeing to pay fees

### ISSUE THREAD

**Issue by adekbadek** [Automattic/newspack-blocks#965]
From [this code review](https://github.com/Automattic/newspack-blocks/pull/928#pullrequestreview-826539501):

> I miss seeing a total of what is going to be charged. When there are additional values it's common to have the total sum next to the donate/pay button.

**Comment by yogeshbeniwal** [comment]
Stripe fee depends on country where account is located, and it varies. It will be better to incorporate logic to calculate fee automatically in code on basis of country. If not possible then option to configure manually in UI.

**Comment by adekbadek** [comment]
Yes, there will be a UI for this. The parameters can be set now via WP options (`newspack_blocks_donate_fee_multiplier`, `newspack_blocks_donate_fee_static`):

https://github.com/Automattic/newspack-blocks/blob/de68cb22402dc67725df9cb8ffe3ad34678108a7/src/blocks/donate/view.php#L162-L163
